package-scripts/datadog-agent/postinst

```diff
@@ -41,15 +41,20 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
               update-rc.d datadog-agent defaults
               echo "Enabling service datadog-agent"
               update-rc.d datadog-agent enable >/dev/null 2>&1
-              # Only add dd-agent user if it doesn't already exist
+              # Only create dd-agent group and/or user if they don't already exist
+              getent group dd-agent >/dev/null || echo "Creating dd-agent group" && addgroup --system dd-agent --quiet
               set +e
               id -u dd-agent >/dev/null 2>&1
               USER_EXISTS=$?
               set -e
               if [ ! $USER_EXISTS -eq 0 ]; then
                   echo "Creating dd-agent user"
-                  adduser --system dd-agent --disabled-login --shell /bin/sh --no-create-home --quiet
+                  adduser --system dd-agent --disabled-login --shell /bin/sh --no-create-home --group --quiet
```

Comment:
you've probably taken a good look at the manpage, so I'm just saying this so we doublecheck - doesn't the `--group` flag take an argument after it? You are using the `--system` flag as well, so I'm not sure about this particular use-case.